### PR TITLE
Counterspell Singapore

### PR DESCRIPTION
**Pull request by emergenitro**
Hi! Just check it out and let me know if there's any issue!

This is for the Counterspell Singapore.

### CODE REVIEW

**Comment by emergenitro** [comment]
Will fix it in 30 mins, missed the pages folder. Sorry!

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import "./index.css";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
```
Comment (samdev-7) [inline src/index.js:4]: Please revert these lines, thanks!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (emergenitro) [reply src/index.js:4]: Sorry! That's because of a formatter I have on. Will change it now

**Review by samdev-7 — APPROVED** [review]

**Comment by emergenitro** [comment]
Thanks! Can't wait for Counterspell Singapore!

**Pull request merged**